Homework/Homework4/Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿using StringCheckLibrary;
+using ArrayProcessing;
+
+namespace Homework4
+{
+    internal class Program
+    {
+        private static void Main(string[] args)
+        {
+            Console.Write("Enter the number of array elements: ");
+            var input = Console.ReadLine();
+            int arrayLenght = int.TryParse(input, out int result) ? result : 0;
+
+            int[] array = ArrayGenerator.GenerateValues(arrayLenght, 1, 26);
+            (int[] evenArray, int[] oddArray) = ArrayProcessor.DivideIntoEvenAndOdd(array);
```

Comment:
Better to create model for this purpose or create method in which we can send input parameter of tje type we need either odd or even

Reply:
fixed in next commit